Wire-iOS/Sources/UserInterface/ConversationList/ArchivedNavigationBar.swift

```diff
@@ -26,12 +26,10 @@ final class ArchivedNavigationBar: UIView {
 
         return view
     }()
-
     let titleLabel: UILabel = {
-        let label = UILabel()
-        label.textColor = .from(scheme: .textForeground, variant: .dark)
-        label.font = .mediumSemiboldFont
-
+        let label = DynamicFontLabel(
+            fontSpec: .mediumSemiboldFont,
+            color: .textBackground)
```

Comment:
I think that in this case you also need to include the variant and set it to dark. Like this:

```suggestion
        let label = DynamicFontLabel(
             fontSpec: .mediumSemiboldFont,
             color: .textBackground
             variant: .dark)
```

Reply:
Yes @agisilaos you are right but take a look at how it's looking on the develop right now:
![image](https://user-images.githubusercontent.com/50630878/170943652-1d2e762d-71f3-43bb-87e4-07e7375e22f3.png)

And after the change that you have suggested it will look like this:
![image](https://user-images.githubusercontent.com/50630878/170944029-878563e9-5852-4150-8cab-b0681bc229c9.png)

Reply:
So there are 2 different colours of the text, I can do as you say but will it be good?